CardForm/src/main/java/com/braintreepayments/cardform/view/RemoveSlashesFilter.java

```diff
@@ -0,0 +1,39 @@
+package com.braintreepayments.cardform.view;
+
+import android.text.InputFilter;
+import android.text.Spanned;
+
+import androidx.annotation.VisibleForTesting;
+
+class RemoveSlashesFilter implements InputFilter {
+
+    static RemoveSlashesFilter newInstance(int maxStringLength) {
+        return newInstance(new StringBuilder(maxStringLength));
+    }
+
+    @VisibleForTesting
+    static RemoveSlashesFilter newInstance(StringBuilder stringBuilder) {
+        return new RemoveSlashesFilter(stringBuilder);
+    }
+
+    private StringBuilder stringBuilder;
```

Comment:
Across the SDK, we put properties at the top of the class.

Reply:
Not sure what the convention is in the style guide we're looking to follow, but just thought it is worth noting.

Reply:
Yeah the google style guide actually leaves this open https://google.github.io/styleguide/javaguide.html#s3.4.2-ordering-class-contents.

For me it made sense to put the properties near the constructor in this case, but I'm gonna stick with the overall project style.